src/pages/Inventory.page.js

```
@@ -1,15 +1,78 @@
+                    description: await this.inventoryItems.nth(randomItemsIndex[i])
+                        .locator(this.inventoryItemDescription).textContent(),
+                    price: await this.inventoryItems.nth(randomItemsIndex[i])
+                        .locator(this.inventoryItemPrice).textContent(),
+                },
+            );
+            await this.addItemToCartById(randomItemsIndex[i]);
+        }
+        return itemsToAddInCart;
+    }
+
+    async openShoppingCart() {
+        await this.shopingCart.click();
+    }
+
+    async filterItemsBy(criteria) {
+        await this.filterItems.selectOption(criteria);
+    }
+
+    async verifyItemsSortedBy(criteria) {
+        let prices;
+        let parsedPrices;
+        let names;
+        if (criteria === 'Price (low to high)') {
+            prices = await this.page.locator(this.inventoryItemPrice).allTextContents();
+            parsedPrices = prices.map((element) => parseFloat(element.replace('$', '')));
+            return parsedPrices.every((value, index, array) => index === 0 || value >= array[index - 1]);
+        }
+        if (criteria === 'Price (high to low)') {
+            prices = await this.page.locator(this.inventoryItemPrice).allTextContents();
+            parsedPrices = prices.map((element) => parseFloat(element.replace('$', '')));
+            return parsedPrices.every((value, index, array) => index === 0 || value <= array[index - 1]);
+        }
+        if (criteria === 'Name (A to Z)') {
+            names = await this.page.locator(this.inventoryItemName).allTextContents();
+            return JSON.stringify(names) === JSON.stringify(names.sort((a, b) => a - b));
+        }
+        if (criteria === 'Name (Z to A)') {
+            names = await this.page.locator(this.inventoryItemName).allTextContents();
+            return JSON.stringify(names) === JSON.stringify(names.sort((a, b) => b - a));
+        }
+        return null;
```

Switch case here? returning **null** bit confusing, better to throw an error if you want to handle this case